examples/torch/requirements.txt

```diff
@@ -7,3 +7,8 @@ mlflow>=1.12.1
 returns==0.14
 opencv-python>=4.4.0.46
 torchvision==0.10.1  # should always match the torch version that is installed via NNCF's setup.py
+efficientnet_pytorch
+
+# Please see
+# https://stackoverflow.com/questions/70520120/attributeerror-module-setuptools-distutils-has-no-attribute-version
+setuptools==59.5.0
```

Comment:
This may be included by https://github.com/openvinotoolkit/nncf/pull/1239.
Do we need to include this in feature branch too?

Reply:
This branch also needs it to run `python examples/experimental/torch/classification/fracbits.py`.